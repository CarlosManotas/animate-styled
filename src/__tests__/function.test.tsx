import AnimateStyled from "..";
import { render } from "@testing-library/react";
import { test, expect } from "vitest";
import "@testing-library/jest-dom";

test("Component should show 'red' text 'Hello World'", () => {
  const { getByText } = render(
    <AnimateStyled name="backInDown" children={undefined} />
  );
  const textElement = getByText("Hello World");
  expect(textElement).toBeInTheDocument();
});
