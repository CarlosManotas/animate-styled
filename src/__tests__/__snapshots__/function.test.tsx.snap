// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Component should match snapshot 1`] = `
<div>
  <div
    class="sc-blHHSb leFnXB"
  >
    <h1>
      Hello World
    </h1>
  </div>
</div>
`;
